src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/Completion/MarkupTransitionCompletionItemProviderTest.cs

```diff
@@ -151,13 +151,8 @@ public void GetCompletionItems_ReturnsEmptyCompletionItemInSingleLineTransitions
         public void GetCompletionItems_ReturnsEmptyCompletionItemInNestedCSharpBlock()
         {
             // Arrange
-            var syntaxTree = CreateSyntaxTree(@"<div>
-@if (true)
-{
-  < @* Should get text completion here *@
-}
-</div>");
-            var location = new SourceSpan(22, 0);
+            var syntaxTree = CreateSyntaxTree("<div>@if (true){< @* Should get text completion here *@}</div>");
+            var location = new SourceSpan(17, 0);
```

Comment:
In the future if you want to do tests like these with multi-lines the way to make them work cross-plat is is to count the newlines and then modify this number (`17`) to be something like `123 + Environment.Newline.Length * 3`. Reason being cross-plat you're not guaranteed to have `\r\n` as your newline sequence as you have in Windows (Linux is `\n`).

Reply:
Ahh, that's why! I was scratching my head trying to figure out why this test case wasn't passing, despite being fine locally. Thanks for clarifying!

Reply:
I'll likely have to apply this change for the other multi-line tests. Let's see if the current build passes...